### ISSUE TITLE
Video player_ Play, volume, Full screen do not have accessible names (ARIA-labels)

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#156]
![image](https://cloud.githubusercontent.com/assets/15111528/11052763/415c803c-87af-11e5-9881-685d33c4f5e0.png)

**Comment by mr-bobz** [comment]
Fixed

**Comment by mr-bobz** [comment]
I upgraded to latest version of mediaelement player (mediaelement-and-player.min.js) before hacking - http://mediaelementjs.com/

That has fixed the aria-labels for all controls

**Comment by christosptr** [comment]
@HerinHentry  and @mr-bobz 

There is still a bit of an issues with the CC caption button. JAWS announce that we need to press the JAWS button + Alt + M to activate the button

If we add an aria-label="press enter to toggle" it will overwrite  all other labels or aria and it will be correct

Steve confirmed last night that aria-label will overwrite the label tag and another attributes like title

**Comment by HerinHentry** [comment]
@mr-bobz : The buttons are also missing alt text.
I havenot done a code inspection for this yet.

![image](https://cloud.githubusercontent.com/assets/15111528/11090267/ba5c596a-88c4-11e5-92fe-c4c8a69d52fa.png)
![image](https://cloud.githubusercontent.com/assets/15111528/11090276/ca50f77c-88c4-11e5-8643-a78301d20580.png)

@christosptr : Thank you for the update.

**Comment by mr-bobz** [comment]
I have added text to the buttons

**Comment by HerinHentry** [comment]
Can close.
